index.d.ts

```diff
@@ -11,6 +11,13 @@ declare module 'spectacle' {
     currentSlide: number;
   }) => React.ReactNode;
 
+  export type useMousetrap = (
+    keybinds: {
+      string: void;
+    }[],
+    deps: any[]
```

Comment:
This is close!

keybinds should be
```
Record<string, (e: ExtendedKeyboardEvent, combo: string) => void>
```

You might need to import `ExtendedKeyboardEvent` from `mousetrap`.